avahi-docker.py

```diff
@@ -78,9 +79,16 @@ def register_avahi():
     containers = c.containers()
     for cont in containers:
         info = c.inspect_container(cont['Id'])
-        ip = info['NetworkSettings']['IPAddress']
-        publish(hostname + '_' + info['Name'][1:]+'.local', ip)
-        publish(info['Name'][1:] + '.' + hostname + '.local', ip)
+        hostnames = (
+            hostname + '_' + info['Name'][1:]+'.local',
+            info['Name'][1:] + '.' + hostname + '.local',
+        )
+        ips = [info['NetworkSettings']['IPAddress']]
+        ip6 = info['NetworkSettings']['GlobalIPv6Address']
```

Comment:
Hi @fsateler thanks for your reply!
I can't get LinkLocalIPv6Address to have any value on my system, but the Link Local address isn't very usefull because you need to specify the network interface which should be used for reaching the host/container. Something like container0.hostname.local%eth0 in my experience not every application support this notation. The "GlobalIPv6Address" is much more usefull.